### PR TITLE
Use `id` *and* `reference_number` to refer to generated documents

### LINKED ISSUE #80: Improved reference numbers
There is a chance we could have multiple summaries for the same appointment. To cover that off @andrewgarner has used the following reference number approach in the documents issued (manually) to date:

```rb
"#{appointment_summary.id}/#{appointment_summary.reference_number}" # 54/12344
```

### PR DESCRIPTION
**Pull request by stevenwilkin**
Resolves #80 

Let this through without tests?

### CODE REVIEW

**Comment by gjvis** [comment]
no tests :scream:

**Comment by andrewgarner** [comment]
bump @stevenwilkin

**Comment by stevenwilkin** [comment]
On it!

**Comment by stevenwilkin** [comment]
come and get it

**Comment by gjvis** [comment]
*you're

File: app/domain/output_document.rb [hunk 1]
```diff
@@ -63,7 +63,7 @@ def lead
   end
 
   def appointment_reference
```
Comment (gjvis) [inline app/domain/output_document.rb]: Perhaps this should be called `output_document_reference`? As it'll (eventually) be related to the output document issue, and not necessarily the appointment reference - right?

Comment (stevenwilkin) [inline app/domain/output_document.rb]: The name was taken from the list of fields Williams Lea are working with

Comment (gjvis) [inline app/domain/output_document.rb]: Ignore me, I misunderstood what we'd done in the initial manual run :blush:

File: spec/domain/output_document_spec.rb [hunk 2]
```diff
@@ -150,6 +152,18 @@ def excludes_all_circumstances
...
+  describe '#appointment_reference' do
+    let(:id) { 'internal-id' }
+
+    before do
+      allow(appointment_summary).to receive(:id).and_return(id)
+    end
+
+    subject { output_document.appointment_reference }
+
+    it { is_expected.to eq("#{id}/#{reference_number}") }
```
Comment (gjvis) [inline spec/domain/output_document_spec.rb]: Can we add a description is make the reason we're doing this clear

**Comment by stevenwilkin** [comment]
Bump

**Comment by andrewgarner** [comment]
:+1:

**Pull request merged**